loader/status.go

```diff
@@ -51,7 +51,7 @@ func (l *Loader) PrintStatus(ctx context.Context) {
 	failpoint.Inject("PrintStatusCheckSeconds", func(val failpoint.Value) {
 		if seconds, ok := val.(int); ok {
 			printStatusInterval = time.Duration(seconds) * time.Second
-			log.Infof("[failpoint] set loader printStatusInterval to %d", seconds)
+			l.tctx.L().Info("set loader printStatusInterval", zap.String("feature", "failpoint"), zap.Int("printStatusInterval", seconds))
```

Comment:
`failpoint` used in many places, Can we add a special filed?

Reply:
logger belong to failpoint's owner of object and failpoint is not a process flow, it's just a feature, right?

Reply:
but we can still create a `zap.String("feature", "failpoint")` field used in all places, right?

Reply:
we need it, we need it and owner function information